prompt_selection.ipynb

```diff
@@ -104,8 +104,19 @@
     "    topic = rl_chain.ToSelectFrom([\"machine learning\"]))\n",
     "\n",
     "print(r[\"response\"])\n",
-    "print(r[\"response_result\"].actions)\n",
-    "print(r[\"response_result\"].context)"
+    "print(r[\"decision_metadata\"].actions)\n",
```

Comment:
was between

- `decision_metadata`
- `action_metadata`
- `decision_details`
- `event_details`


open to suggestions